<launch>
  <arg name="group_name" />
  <arg name="n_board" default="6" />
  <arg name="duration" default="0.1" />
  <arg name="touch_prx_threshold" default="500" />

  <group ns="$(arg group_name)">
    <node name="rmui_node" pkg="rmui_drivers" type="rmui_node.py"
          output="screen">
      <rosparam subst_value="true">
        n_board: $(arg n_board)
        duration: $(arg duration)
        touch_prx_threshold: $(arg touch_prx_threshold)
      </rosparam>
    </node>
  </group>
</launch>
